services/dynamic_addressbook_service.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+option java_package = "com.hederahashgraph.service.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.token">>> This comment is special code for setting PBJ Compiler java package
+
+import "query.proto";
+import "response.proto";
+import "transaction_response.proto";
+import "transaction.proto";
+
+/**
+ * Transactions for Dynamic Address Book Service
+ */
+service DynamicAddressBookService {
+    /**
+     * Add a new node to address book by submitting the transaction
+     */
+    rpc addNode (Transaction) returns (TransactionResponse);
+
+    /**
+     * Remove a node from address book by submitting the transaction
+     */
+    rpc removeNode (Transaction) returns (TransactionResponse);
```

Comment:
I believe we use mostly "delete" in our API and should be consistent.

Reply:
Initially, I used deleteNode. Then I noticed in our HIP, it says Remove Node. Also it's mentioned in the call, we should not delete node. Removed nodes will be kept somewhere. So I used removeNode.

Reply:
I agree with Michael. Whether we immediately purge it from all memory and disk or wait a while is an implementation detail. Consistency in naming with the rest of our API is the most important.

Reply:
changed to deleteNode.